http/file_server.ts

```diff
@@ -454,6 +455,7 @@ export interface ServeDirOptions {
  *
  * Optionally you can pass `urlRoot` option. If it's specified that part is stripped from the beginning of the requested pathname.
  *
+ * @example
  * ```ts
```

Comment:
text below this codeblock should be moved to be above it, but still part of title